### PR TITLE
Created --version command and added tests

### PR DESCRIPTION
**Pull request by MordoJ**

### CODE REVIEW

**Review by clemdiep-heig — CHANGES_REQUESTED** [review]
Use `@Option` instead of a subcommand https://picocli.info/#_options_and_parameters

Put the `version.txt` file in the `resources` or directly in the code and we will see later to change it

File: src/main/java/ch/heigvd/statique/Statique.java [hunk 1]
```diff
@@ -28,7 +30,17 @@ public static void main(String... args) {
 
   @Override
   public Integer call() throws Exception {
-    CommandLine.usage(this, System.out);
+    if(version) {
+      try {
+        String v = Files.readString(Path.of("src/main/resources/about/version.txt"));
```
Comment (clemdiep-heig) [inline src/main/java/ch/heigvd/statique/Statique.java:35]: Use this to read a file from resource directory, use this

```suggestion
      try (InputStream inputStream = getClass().getClassLoader().getResourceAsStream("about/version.txt")) {
        String v = new String(inputStream.readAllBytes());
```

File: src/main/java/ch/heigvd/statique/Statique.java [hunk 2]
```diff
@@ -1,9 +1,9 @@
 package ch.heigvd.statique;
 
-import ch.heigvd.statique.commands.Build;
-import ch.heigvd.statique.commands.Clean;
-import ch.heigvd.statique.commands.Init;
-import ch.heigvd.statique.commands.Serve;
+import ch.heigvd.statique.commands.*;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
```
Comment (clemdiep-heig) [inline src/main/java/ch/heigvd/statique/Statique.java:6]: Need to add this import

```suggestion
import java.io.InputStream;
```

**Review by clemdiep-heig — CHANGES_REQUESTED** [review]
This correction are working on my environment

**Review by MordoJ — COMMENTED** [review]
I've added Clement's corrections. The code I committed was working in my environment but not in his.

**Review by clemdiep-heig — APPROVED** [review]
🍻

**Review by dylanrodr-heig — APPROVED** [review]

**Pull request merged**